### ISSUE TITLE
op.fi cookie banner

### ISSUE THREAD

**Issue by 07416** [finnish-easylist-addition/finnish-easylist-addition#253]
### List the website(s) you're having issues:

`https://www.op.fi/`

### What happens?

a cookie banner:

![Screenshot_2020-06-11 OP-verkkopalvelu, op fi, osuuspankkien verkkopankki](https://user-images.githubusercontent.com/14923168/84415683-cbb5b880-ac1b-11ea-9cb9-cc2b9d8b38da.png)

### Other details:
`op.fi` has no entries on the list.

**Comment by peace2000** [comment]
Hi! The scope of this list is limited to ads. There is specific list that will block cookie banners - Easylist Cookie. Those can be reported here: https://github.com/easylist/easylist/issues

I'll make a pull request to add blocking for this. Meanwhile, you can block that by adding this to your own filters: ||tags.op-palvelut.fi/ocm/opfi/consent.js$script,domain=www.op.fi

**Comment by peace2000** [comment]
https://github.com/easylist/easylist/pull/5543

**Comment by 07416** [comment]
The element title is indeed `##.ocm-container--banner`, sorry for forgetting this from the report.

- - -

@peace2000 [The annoyance list](https://raw.githubusercontent.com/finnish-easylist-addition/finnish-easylist-addition/master/annoyances/Finland_annoyances.txt) has at least one cookie rule:

`lehtodigital.fi###_kmg_cookie_accept`

I thought that cookie elements are within the scope of the list. I also wasn't sure if Easylist accepts foreign websites in any of its lists, but they seem to.

**Comment by 07416** [comment]
> I'll make a pull request to add blocking for this. Meanwhile, you can block that by adding this to your own filters: `||tags.op-palvelut.fi/ocm/opfi/consent.js$script,domain=www.op.fi`

Blocking the script would indeed by optimal, Easylist doesn't support some of the advanced rules since they want to maintain Adblock Plus compatibility (?).

It'd be very nice if this list included cookie warning banners.

**Comment by peace2000** [comment]
Easylist has a section for non-english sites in the Easylist Cookie and in the Easylist Privacy lists.

Blocking network requests is a normal, widely supported feature. ABP can handle that as well.

That annoyance list has been unmaintained for a long time. When it was actively maintained, it was maintained by the other maintainer of the Finnish Easylist but he has been busy lately. The only edit I have made to the Annoyance list is that I removed a space from there. I don't feel that it's useful to maintain a list that is not in wide use. People also have different opinions what is considered an annoyance.

Also when there is a major list that blocks cookie banners, I don't feel that is necessary to maintain another list for that, especially when it accepts rules for the Finnish sites as well.
